x-pack/platform/plugins/shared/integration_assistant/server/integration_builder/build_integration.test.ts

```diff
@@ -245,7 +237,7 @@ describe('renderPackageManifestYAML', () => {
       logo: 'some-logo.png',
       dataStreams: [
         {
-          name: 'data-stream-1',
+          name: 'data-stream-one',
```

Comment:
this is now an invalid name as we do not accept hyphens. Even if it does not fail the test we should probably change the names to valid ones, for consistency